nemo/core/classes/exportable.py

```diff
@@ -285,88 +172,23 @@ def supported_export_formats(self):
         """Implement this method to return a set of export formats supported. Default is all types."""
         return set([ExportFormat.ONNX, ExportFormat.TORCHSCRIPT])
 
-    @staticmethod
-    def _extract_dynamic_axes(name: str, ntype: NeuralType):
-        """
-        Implement this method to provide dynamic axes id for ONNX export.
-        By default, this method will extract BATCH and TIME dimension ids from each provided input/output name argument.
-
-        For example, if module/model accepts argument named "input_signal" with type corresponding to [Batch, Time, Dim]
-        shape, then the returned result should contain "input_signal" -> [0, 1] because Batch and Time are dynamic axes
-        as they can change from call to call during inference.
-
-        Args:
-            name: Name of input or output parameter
-            ntype: Corresponding Neural Type
-
-        Returns:
-
-        """
-        dynamic_axes = defaultdict(list)
-        if type(ntype) in (list, tuple):
-            ntype = unpack_nested_neural_type(ntype)
-
-        if ntype.axes:
-            for ind, axis in enumerate(ntype.axes):
-                if axis.kind in [AxisKind.Batch, AxisKind.Time, AxisKind.Width, AxisKind.Height]:
-                    dynamic_axes[name].append(ind)
-        return dynamic_axes
-
     def _prepare_for_export(self, **kwargs):
         """
         Override this method to prepare module for export. This is in-place operation.
         Base version does common necessary module replacements (Apex etc)
         """
-        replace_1D_2D = kwargs.get('replace_1D_2D', False)
-        replace_for_export(self, replace_1D_2D)
+        replace_for_export(self)
 
     def _export_teardown(self):
         """
         Override this method for any teardown code after export.
         """
         pass
 
-    def _wrap_forward_method(self):
```

Comment:
Without these methods RNNT export will fail. Also why such a heavy refactor back to monolithic design of export? It will make future models harder to deal with